sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 #### Key Bugs Fixes
 * Added `decodeTime` in `CosmosDiagnostics` - See [PR 22808](https://github.com/Azure/azure-sdk-for-java/pull/22808)
 
+#### Features Added
```

Comment:
Thanks for adding this, however, `Features Added` section should come above `Key Bug Fixes` section.